rootfs/scheduler/__init__.py

```diff
@@ -870,9 +884,9 @@ def _wait_until_pods_terminate(self, namespace, labels, current, desired):
 
             time.sleep(1)
 
-        logger.debug("{} pods in namespace {} are terminated".format(delta, namespace))  # noqa
+        logger.debug("{} pods in namespace {} are terminated".format(delta, namespace))
```

Comment:
ah, yea. forgot the name and I guess I forgot that I forgot. good to know RE the line length check